src/server/standalone.js

```diff
@@ -642,9 +642,7 @@ function StandAloneServer(gmeConfig) {
     // Basic authentication
 
     logger.debug('creating login routing rules for the static server');
-    __app.get('/', ensureAuthenticated, function (req, res) {
-        expressFileSending(res, __clientBaseDir + '/index.html');
-    });
+    __app.get('/', ensureAuthenticated, Express.static(__clientBaseDir));
```

Comment:
Yepp, the default behavior of static is that it enables directory lookup, and the default file is index.html